web-animations-1/Overview.bs

```diff
@@ -2465,123 +2465,203 @@ Animation effects {#animation-effects}
-             on the endpoints of the active interval">
+    <img src="images/time-spaces.svg" width="600"
+         alt="A comparison of local time, active time, and iteration time.
+              Local time crosses zero at the start time of the animation,
+              and rises linearly.
+              Active time is zero until the start of the active interval,
+              at which point it rises linearly to the end time,
+              beyond which it remains at its maximum value.
+              Iteration time is likewise zero until the start of the active interval,
+              and rises linearly,
+              but restarts at zero at the start of every new iteration,
+              and remains at its halfway point from the end time onward.">
     <figcaption>
-      Examples of the effect of the [=start delay=]
-      on the endpoints of the [=active interval=].<br>
-      (a) An animation effect with no delay;
-      the [=start time=] and beginning of the [=active interval=]
-      are coincident.<br>
-      (b) An animation effect with a positive delay;
-      the beginning of the [=active interval=]
-      is deferred by the delay.<br>
-      (c) An animation effect with a negative delay;
-      the beginning of the [=active interval=]
-      is brought forward by the delay.
+      A comparison of [=local time=], [=active time=], and [=iteration time=]
+      for an animation with an [=iteration duration=] of 1s
+      and an [=iteration count=] of 2.5.
+      The [=animation/start time=] defined by the [=animation=]
+      and [=animation effect/end time=] defined by the [=animation effect=]
+      are also annotated.
     </figcaption>
   </figure>
 
-  An [=end delay=] can also be specified
-  but is primarily only of use when sequencing animations.
+  In addition to these time spaces
+  we can also refer to the <dfn noexport>document time space</dfn>,
+  which is time space of the [=time values=]
+  of the [=default document timeline=] of the {{Document}}
+  of the [=current global object=].
 
+  Note: While the time spaces themselves are not bounded,
+  Web Animations defines [=active time=] and the [=iteration progress=]
+  such that they are clamped to a set range as shown in the diagram.
+  For example, whilst a time of -1 second
+  is a valid time in <em>active time space</em>,
+  the procedure for calculating the [=active time=]
+  defined in [[#calculating-the-active-time]]
+  will never return a negative value.
 </div>
 
-  [=Animation effects=] define an <dfn>active interval</dfn>
+### The active interval, duration, and delays ### {#the-active-interval}
+
+  [=Animation effects=] define an <dfn>active interval</dfn>,
   which is the period of time during which the effect
-  is scheduled to produce its effect--
-  with the exception of [=fill modes=],
-  which apply outside the [=active interval=].
+  is scheduled to produce its effect
+  (excepting [=fill modes=],
+  which apply outside the [=active interval=]).
```

Comment:
I wonder if this should be a parenthetical. I think the information is pretty central to the definition?

Reply:
Ok, I trust your instincts for this.